build-livecd-root

```diff
@@ -21,8 +21,9 @@ fi
 
 trap cleanup EXIT
 
+KEEP_TMPDIR=yes
```

Comment:
Ah sorry,

That was added by me for some debugging. That was pushed unintentionally.
I'll remove that.

-PP

On Mon, Aug 3, 2015 at 7:39 PM, Lukáš Zapletal <notifications@github.com>
wrote:

> In build-livecd-root
> <https://github.com/theforeman/foreman-discovery-image/pull/20#discussion_r36088114>
> :
>
> > @@ -21,8 +21,9 @@ fi
> >
> >  trap cleanup EXIT
> >
> > +KEEP_TMPDIR=yes
>
> I'd rather keep this to no as a default value. I build ten images a day
> and /tmp is by default a memory mapped folder...
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/theforeman/foreman-discovery-image/pull/20/files#r36088114>
> .
>